oauth2c.gemspec

```diff
@@ -45,14 +47,14 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "http", "~> 2.0"
-  spec.add_runtime_dependency "jwt", "~> 1.5"
+  spec.add_runtime_dependency "http"
+  spec.add_runtime_dependency "jwt"
 
-  spec.add_development_dependency "bundler", "~> 1.14"
+  spec.add_development_dependency "bundler"
   spec.add_development_dependency "pry"
   spec.add_development_dependency "byebug"
-  spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "redis", "~> 3.0"
-  spec.add_development_dependency "rspec", "~> 3.0"
-  spec.add_development_dependency "webmock", "~> 2.0"
```

Comment:
I don't think this is a good idea, specially for the runtime dependencies, and we can't guarantee compatibility with different major versions.

@cv let's adjust the requirements, but not get rid of it.